internal/service/databaseuser/resource_database_user.go

```diff
@@ -231,6 +231,8 @@ func (r *databaseUserRS) Create(ctx context.Context, req resource.CreateRequest,
 		return
 	}
 
+	resp.Diagnostics.AddWarning("If the password value will be managed externally it is advised to remove the attribute", "More details can be found in resource documentation under the 'password' attribute")
```

Comment:
is this warning shown always or it has some condition?

Reply:
will always be shown, not really any particular condition we can use because password will always be defined for create operation (fails in API otherwise)